nestkernel/connection_manager.cpp

```diff
@@ -643,19 +643,22 @@ nest::ConnectionManager::disconnect( Node& target,
     // get the ConnectorBase corresponding to the source
     ConnectorBase* conn =
       validate_pointer( validate_source_entry_( target_thread, sgid, syn_id ) );
-    ConnectorBase* c =
-      kernel()
-        .model_manager.get_synapse_prototype( syn_id, target_thread )
-        .delete_connection( target, target_thread, conn, syn_id );
-    if ( c == 0 )
+    if ( conn != 0 )
```

Comment:
Could we simplify code a little by writing
```C++
if ( conn == 0 )
{
  return;
}
```
and then the rest outside any block?